TL improvement & official TL import (#539)
07th-mod/higurashi-console-arcs: Update/omot_010_4.txt

			NULL, " ...Rena, Rina Mamiya is a con artist.", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100854", 256, TRUE);
	OutputLine(NULL, "男と組んで、結婚詐欺を働いてる」",

> 			NULL, " She gets together with a man and uses them for marriage fraud scams.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " She gets together with men and uses them for marriage fraud scams.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100855", 256, TRUE);
	OutputLine(NULL, "「リナは最近、雛見沢にカモを見つけたって言いふらしてるらしい。",

> 			NULL, "\"They say Rina keeps bragging that she recently found another easy mark in Hinamizawa.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"They say Rina has been bragging recently about finding another easy mark in Hinamizawa.", Line_WaitForInput);
```

//r梨花ちゃんはため息をついて、肩を落とす。…実際俺もここまで拒絶されるとは思っていなかったので、落胆の気持ちを禁じえなかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "梨花ちゃんはため息をついて、肩を落とす。…実際俺もここまで拒絶されるとは思っていなかったので、落胆の気持ちを禁じえなかった。",

> 			NULL, "Rika-chan sighed and slumped her shoulders. ...Honestly, even I didn't expect us to be so outright rejected, so I could help but feel disappointed.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Rika-chan sighed and slumped her shoulders. ...Honestly, even I didn't expect us to be so outright rejected, so I couldn't help but feel disappointed.", Line_Normal);
```
